src/nunit.analyzers/Extensions/AttributeArgumentTypedConstantExtensions.cs

```diff
@@ -26,32 +26,37 @@ internal static class AttributeArgumentTypedConstantExtensions
 
         // Intrinsic type converters for types
         // https://github.com/dotnet/runtime/blob/master/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs
-        // For converters that exist in .NET Standard 1.6 we use the converter.
+        // For converters that exist in .NET Standard 2.0 we use the converter.
         // Otherwise, we assume that the converter exists and it can convert the value.
-        private static readonly List<(Type type, Lazy<TypeConverter>? typeConverter)> IntrinsicTypeConverters =
+        private static readonly List<(string fullName, Lazy<TypeConverter>? typeConverter)> IntrinsicTypeConverters =
             new()
             {
-                (typeof(bool), new Lazy<TypeConverter>(() => new BooleanConverter())),
```

Comment:
Updated mapping per TypeConverter updates:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs#L154

Had to update it to string from Type, as netstandard2.0 does not have  DateTime/DateOnly/etc types.